src/main/java/com/thealgorithms/backtracking/subset(Skienaform).java

```diff
@@ -0,0 +1,46 @@
+package com.thealgorithms.backtracking;
+
+class Subset{
+	static void Backtracking(int a[],int k,int n,int input[]){
```

Comment:
Please give descriptive names to the parameters instead of `a`, `k`, `n`, and `input`. And the function name needs to be changed as well.

Reply:
But the function names are same as of skiena pseudocode which I was trying to replicate

Reply:
If I want to use this function, I wouldn't know what `Subset.Backtracking` does and what are `a, k, n, input` that it accepts. Function signature must be self-explanatory.